packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -20,6 +20,9 @@ export class AtomStepsModal {
   @Prop() trigger?: string
   @Prop() stepsTitles: string
   @Prop({ mutable: true }) isOpen = false
+  @Prop() primaryText? = 'Próximo'
+  @Prop() secondaryText? = 'Voltar'
```

Comment:
block:

1. We need to keep always in English
2. We don't need to add a default value for it, it is a optional attribute